src/util/rebind.js

```diff
@@ -49,16 +49,35 @@ export function rebindAll(target, source, prefix, exclusions) {
         exclusions = args.slice(3);
     }
 
-    exclusions.forEach(function(property) {
+    var stringExclusions = exclusions.filter(function(exclusion) {
```

Comment:
I was going to make a few suggestions here then got carried away and [created a PR instead](https://github.com/ColinEberhardt/d3-financial-components/pull/1)